app/pages/preRequisites/AddCountryOfDestinationPage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.preRequisites
+
+import controllers.preRequisites.routes
+import models.{Mode, OptionalBoolean, UserAnswers}
+import pages.QuestionPage
+import pages.sections.PreRequisitesSection
+import play.api.libs.json._
+import play.api.mvc.Call
+
+case object AddCountryOfDestinationPage extends QuestionPage[OptionalBoolean] {
```

Comment:
Are we missing cleanup logic here?
<img width="683" alt="Screenshot 2024-09-20 at 08 41 10" src="https://github.com/user-attachments/assets/2d539fd8-cfae-498e-9b79-da8ce76fd862">

Reply:
There's follow up tickets for the nav and clean up logic (CTCP-5722 + CTCP-5723)